### PR TITLE
Fixed issue #18678: missing translation in bread crumbs.

### PR DESCRIPTION
**Pull request by gabrieljenik**
Translation function added to title property.

### CODE REVIEW

File: application/controllers/SurveyAdministrationController.php [hunk 1]
```diff
@@ -1945,7 +1945,7 @@ public function actionRendersidemenulink($surveyid, $subaction)
         $aData['action'] = $menuEntry->action;
         $aData['entryData'] = $menuEntry->attributes;
         $aData['dateformatdetails'] = getDateFormatData(Yii::app()->session['dateformat']);
-        $aData['subaction'] = $menuEntry->title;
+        $aData['subaction'] = gT($menuEntry->title);
```
Comment (Shnoulle) [inline application/controllers/SurveyAdministrationController.php:1948]: Came from model , must not translate if [$menuEntry->language](https://github.com/LimeSurvey/LimeSurvey/blob/master/application/models/SurveymenuEntries.php#L21) == admin language

Must translate only if $menuEntry->language is en (or [en-GB](https://github.com/LimeSurvey/LimeSurvey/blob/dfa8c7d07cbfc6f2987dbe248afe4824092df8be/application/core/LsDefaultDataSets.php#L39) ) .

**Review by Shnoulle — COMMENTED** [review]
Came from DB : translation only if en or en-GB

**Review by Shnoulle — APPROVED** [review]

**Comment by Shnoulle** [comment]
PS : it must be fixed elsewhere (i ùmust report broken HTML allowed in attribute help too)

**Comment by tiborpacalat** [comment]
I haven't encountered this issue in LS 6.x so you need to rebase it to 5.x

6.x in Croatian (before fix):
Screenshot 2023-04-21 at 16 00 51

5.x in Croatian (before fix):
Screenshot 2023-04-21 at 16 00 04

**Comment by gabrieljenik** [comment]
New PR: https://github.com/LimeSurvey/LimeSurvey/pull/3073

**Pull request closed without merging**